spec/ddtrace/runtime/cgroup_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'spec_helper'
-require 'ddtrace/runtime/cgroup'
+require 'datadog/core/environment/cgroup'
 
 # rubocop:disable Layout/LineLength
-RSpec.describe Datadog::Runtime::Cgroup do
+RSpec.describe Datadog::Core::Environment::Cgroup do
   describe '::descriptors' do
     subject(:descriptors) { described_class.descriptors }
```

Comment:
This spec should be moved to match the library folder structure. There's more specs that are now mismatched with the library folder (not on datadog, not under core), I suggest doing a full pass of every file changed under lib to make sure its spec is on the correct place now.